jest.config.js

```diff
@@ -1,8 +1,12 @@
+const esModule = '@iconscout/react-unicons';
+
 module.exports = {
   verbose: false,
   transform: {
     '^.+\\.(ts|tsx|js|jsx)$': 'ts-jest',
+    [`(${esModule}).+\\.js$`]: 'babel-jest',
```

Comment:
Is this working fine? Wonder why it did not work for me.

Reply:
Worked like a charm.